src/verification/mod.rs

```diff
@@ -140,6 +141,15 @@ where
                     .cloned()
                     .collect(),
             ),
+            allowed_jose_types: Some(HashSet::from([
+                JsonWebTokenType::new("application/jwt".to_string())
+                    .normalize()
+                    .unwrap(), // used by many IdP, but not standardized
```

Comment:
since this is the production code path, let's use `.expect("application/jwt should be a valid JWT type")` instead of `.unwrap()` so that the panic message is at least somewhat informative if our assumptions are wrong